GLTFSDK/Inc/GLTFSDK/GLTF.h

```diff
@@ -118,8 +109,9 @@ namespace Microsoft
 
         enum ProjectionType
         {
-            PERSPECTIVE,
-            ORTHOGRAPHIC
+            PROJECTION_UNKNOWN = 0,
```

Comment:
Why is this needed?

Reply:
It isn't. I was experimenting with how to handle invalid projection types but ended up just throwing an exception instead. Will remove